test/long-lines/signature.py

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 #
-# __COPYRIGHT__
```

Comment:
Doesn't seem to have much to do with this change.   There should still be a Copyright statement as it's part of the license template, it just doesn't need to be replaceable.  `# Copyright The SCons Foundation` or some such.